server/routes/referrals/referralsController.ts

```diff
@@ -243,6 +243,31 @@ export default class ReferralsController {
     ControllerUtils.renderWithLayout(res, view, serviceUser)
   }
 
+  async viewCohortComplexityLevel(req: Request, res: Response): Promise<void> {
+    const { accessToken } = res.locals.user.token
+    const referral = await this.interventionsService.getDraftReferral(accessToken, req.params.id)
+
+    if (!referral.serviceCategoryIds) {
+      throw new Error('Attempting to view complexity level without service categories selected')
+    }
+
+    const serviceCategoryId = referral.serviceCategoryIds.find(id => id === req.params.serviceCategoryId)
+
+    if (!serviceCategoryId) {
```

Comment:
nit: seems a roundabout way to do array.includes?

Reply:
It really does - didn't realise I'd missed that, thanks